presto-main/src/main/java/com/facebook/presto/server/ClusterStatsResource.java

```diff
@@ -22,17 +22,20 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import javax.annotation.security.RolesAllowed;
 import javax.inject.Inject;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
+import static com.facebook.presto.server.security.RoleType.ADMIN;
 import static java.util.Objects.requireNonNull;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 @Path("/v1/cluster")
+@RolesAllowed(ADMIN)
```

Comment:
This is used by the UI which is generally visible to everyone - should this be just `ADMIN`? I believe that we should add `USER` here as well. Thoughts @arhimondr ?